### ISSUE TITLE
Quoted vs Unquoted booleans 

### ISSUE THREAD

**Issue by stefanmortensen** [atomia/puppet-atomia#85]
A standard is needed how to support booleans properly. The issue at hand is that when using the puppet module in conjunction with the new installation gui we can only export data as strings which leads to booleans (true/false) being interpreted by Puppet as 'true'/'false'. This is not very pretty and causes confusion when using yaml as a backend to Hiera. A few different solutions to this problem:

* Allow both unquoted and quoted booleans. We could have a comparison function that we use to do all comparisons to make this a a bit prettier

* Figure out a way to export data from the database to puppet as booleans

* Use 0/1 instead of true/false everywhere

Bottom line is that currently this is not consistent within all manifests.

**Comment by stefanmortensen** [comment]
Resolved by implementing option 3
